matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/contentscanning/ContentScannerService.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 New Vector Ltd - All Rights Reserved
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ */
+
+package org.matrix.android.sdk.api.session.contentscanning
+
+import androidx.lifecycle.LiveData
+import org.matrix.android.sdk.api.MatrixCallback
+import org.matrix.android.sdk.api.util.Optional
+import org.matrix.android.sdk.internal.crypto.attachments.ElementToDecrypt
+
+interface ContentScannerService {
+
+    val serverPublicKey: String?
+
+    fun getContentScannerServer(): String?
+    /**
+     * Get the current public curve25519 key that the AV server is advertising.
+     * @param callback on success callback containing the server public key
+     */
+    fun getServerPublicKey(forceDownload: Boolean = false, callback: MatrixCallback<String?>)
```

Comment:
We do not use `MatrixCallback` nowadays, but `suspend fun`. Take the opportunity to migrate the API? See #2449 for more details